store/mongo/migration_1_5_0_test.go

```diff
@@ -0,0 +1,202 @@
+// Copyright 2018 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+package mongo
+
+import (
+	"context"
+	"crypto/sha256"
+	"testing"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/identity"
+	"github.com/mendersoftware/go-lib-micro/mongo/migrate"
+	ctxstore "github.com/mendersoftware/go-lib-micro/store"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/mendersoftware/deviceauth/model"
+)
+
+func TestMigration_1_5_0(t *testing.T) {
+	pubKey := `-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzogVU7RGDilbsoUt/DdH
+VJvcepl0A5+xzGQ50cq1VE/Dyyy8Zp0jzRXCnnu9nu395mAFSZGotZVr+sWEpO3c
+yC3VmXdBZmXmQdZqbdD/GuixJOYfqta2ytbIUPRXFN7/I7sgzxnXWBYXYmObYvdP
+okP0mQanY+WKxp7Q16pt1RoqoAd0kmV39g13rFl35muSHbSBoAW3GBF3gO+mF5Ty
+1ddp/XcgLOsmvNNjY+2HOD5F/RX0fs07mWnbD7x+xz7KEKjF+H7ZpkqCwmwCXaf0
+iyYyh1852rti3Afw4mDxuVSD7sd9ggvYMc0QHIpQNkD4YWOhNiE1AB0zH57VbUYG
+UwIDAQAB
+-----END PUBLIC KEY-----`
+
+	ts := time.Now()
+
+	ctx := identity.WithContext(context.Background(), &identity.Identity{
+		Tenant: "foo",
+	})
+	db.Wipe()
+	db := NewDataStoreMongoWithSession(db.Session())
+	s := db.session
+
+	// prep base version
+	mig110 := migration_1_1_0{
+		ms:  db,
+		ctx: ctx,
+	}
+	mig120 := migration_1_2_0{
+		ms:  db,
+		ctx: ctx,
+	}
+	mig130 := migration_1_3_0{
+		ms:  db,
+		ctx: ctx,
+	}
+	mig140 := migration_1_4_0{
+		ms:  db,
+		ctx: ctx,
+	}
+	err := mig110.Up(migrate.MakeVersion(1, 1, 0))
+	assert.NoError(t, err)
+	err = mig120.Up(migrate.MakeVersion(1, 2, 0))
+	assert.NoError(t, err)
+	err = mig130.Up(migrate.MakeVersion(1, 3, 0))
+	assert.NoError(t, err)
+	err = mig140.Up(migrate.MakeVersion(1, 4, 0))
+	assert.NoError(t, err)
+
+	devs := []model.Device{
+		{
+			Id:              "1",
```

Comment:
model.Device contains also PubKey field. It would be good to check if it's still valid after migration.

Reply:
fixed